resources/app.yaml

```diff
@@ -106,10 +106,10 @@ nodeProfiles:
 hooks:
   install:
     job: file://install.yaml
-  postInstall:
-    job: file://postInstall.yaml
-  preUpdate:
-    job: file://preUpdateHook.yaml
+  #postInstall:
```

Comment:
remove these?

Reply:
They are intended to be reimplemented in another PR